docs/observe/logstash-kibana.md

```diff
@@ -0,0 +1,184 @@
+# Logging using logstash and Kibana
+
+Ballerina has a log module for logging to the console. You can import the `ballerina/log` module and start logging. The following section will describe how to search, analyze, and visualize logs in real time using [Elastic Stack](https://www.elastic.co/).
+
+To understand how you can track logging for Ballerina services, let’s consider a service that converts JSON to XML.
+
+![alt text](../../assets/img/logstash-kibana.png)
```

Comment:
Can we move the request and response arrows closer to make it visually comfortable?

Shall we number and describe the data flow paths in the diagram for clarity?

Reply:
Will send this in a subsequent PR.